docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1018,7 +1021,36 @@ xrdsst service disable-descriptions --ss <SECURITY_SERVER_NAME> --client <CLIENT
 * <SERVICE_DESCRIPTION_ID> id of the service description, e.g. 123, multiple values can also be given, separated by comma, e.g. 123,456
 * <NOTICE> disabling notice, e.g. "Not used"
 
-##### 4.2.7.11 Service apply
+##### 4.2.7.11 Service list access rights for services
+
+* Access rights: XROAD_SYSTEM_ADMINISTRATOR and XROAD_SERVICE_ADMINISTRATOR
+
+There are no configuration parameters involved, command line arguments are used instead
+
+Listing access rights for services for client's service descriptions can be done with:
+```
+xrdsst service list-access --client <CLIENT_ID> --description <SERVICE_DESCRIPTION_ID>
+```
+* <CLIENT_ID> id of the client, e.g. DEV:GOV:1234:TEST
+* <SERVICE_DESCRIPTION_ID> id of the service description, e.g. 123, multiple values can also be given, separated by comma, e.g. 123,456
+
+##### 4.2.7.12 Service delete access rights for services
+
+* Access rights: XROAD_SYSTEM_ADMINISTRATOR and XROAD_SERVICE_ADMINISTRATOR
+
+There are no configuration parameters involved, command line arguments are used instead
+
+Deleting access rights for services for client's service descriptions can be done with:
+```
+xrdsst service delete-access --ss <SECURITY_SERVER_NAME> --client <CLIENT_ID> --description <SERVICE_DESCRIPTION_ID> --service <SERVICE_ID> --sclient <SERVICE_CLIENT_ID>
+```
+* <SECURITY_SERVER_NAME> name of the security server, e.g. ss1
+* <CLIENT_ID> id of the client, e.g. DEV:GOV:1234:TEST
+* <SERVICE_DESCRIPTION_ID> id of the service description, e.g. 123, multiple values can also be given, separated by comma, e.g. 123,456
+* <SERVICE_ID> id of the service, e.g. DEV:GOV:1234:TEST:Petstore, multiple values can also be given, separated by comma, e.g. DEV:GOV:1234:TEST:Petstore,DEV:GOV:1234:TEST:Petstore2
+* <SERVICE_CLIENT_ID> id of the service description, e.g. DEV:security-server-owners, multiple values can also be given, separated by comma, e.g. DEV:GOV:1234:TEST,DEV:security-server-owners
+
```

Comment:
Here we could describe how the command works when multiple clients, services, or descriptions are entered. Because it is not entirely clear how it works, from what I see it goes through everything and if it does not find the sclient it skipped it without putting any message

Reply:
Refactored the code and updated the documentation, all operations now consist only a single parameter allowing multiple values so it should not be confusing anymore